sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -324,14 +324,14 @@ case class AttributeReference(
  * A place holder used when printing expressions without debugging information such as the
  * expression id or the unresolved indicator.
  */
-case class PrettyAttribute(
+case class PrettyNamedExpression(
```

Comment:
This rename looks a bit weird to me because now `PrettyNamedExpression` extends `Attribute`.

Reply:
Based on your suggestion, I modified a bit.